### ISSUE TITLE
garmin connect refuse to import a gpx file exported from wanderer

### ISSUE THREAD

**Issue by CPlusPlus17** [Flomp/wanderer#128]
Import gives error, asked chatgpt about it:

_Yes, the XML snippet you provided is mostly valid, but there is a small issue with your xmlns attribute. Specifically, the xmlns attribute appears twice in the <gpx> tag, which is incorrect in XML. You only need one xmlns declaration for the gpx namespace. You can safely remove the duplicate one._

Modified

```
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<gpx version="1.1" creator="wanderer"
	xmlns="http://www.topografix.com/GPX/1/1"
	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.topografix.com/GPX/1/1 http://www.topografix.com/GPX/1/1/gpx.xsd"
	xmlns="http://www.topografix.com/GPX/1/1">
	<metadata>
		<name>Biwak Test</name>
		<desc/>
		<time>2024-11-15T00:00:00Z</time>
		<keywords>jlrl2pyyj8hpjkh, </keywords>
		<author>
			<name>tlfrhfdi2gwfys7</name>
			<email>manuel.gysin@protonmail.com</email>
		</author>
	</metadata>
```

to

```
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<gpx version="1.1" creator="wanderer"
	xmlns="http://www.topografix.com/GPX/1/1"
	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.topografix.com/GPX/1/1 http://www.topografix.com/GPX/1/1/gpx.xsd">
	<metadata>
		<name>Biwak Test</name>
		<desc/>
		<time>2024-11-15T00:00:00Z</time>
		<keywords>jlrl2pyyj8hpjkh, </keywords>
		<author>
			<name>tlfrhfdi2gwfys7</name>
			<email>manuel.gysin@protonmail.com</email>
		</author>
	</metadata>
```

Import works now.

**Comment by Flomp** [comment]
Now fixed in v0.12.0

**Comment by CPlusPlus17** [comment]
Thanks you!
